src/components/views/avatars/MemberAvatar.js

```diff
@@ -65,9 +69,19 @@ module.exports = React.createClass({
 
         var {member, ...otherProps} = this.props;
 
+        var onClick = null;
```

Comment:
some uses of MemberAvatar set `props.onClick` to other things; you'll need to pass that down as the default, and probably exclude it from `otherProps` above.